### PR TITLE
Feature #3: Add A Grade - Kimberly Roque

### PR DESCRIPTION
**Pull request by billy2anne**
Screen Shot 2020-05-12 at 11 12 44 PM

### CODE REVIEW

File: features/components/app.js [hunk 1]
```diff
@@ -29,5 +33,34 @@ class App {
 
   start(){
     this.getGrades();
+    this.gradeForm.onSubmit(this.createGrade);
+  }
+
+  createGrade(name, course, grade){
+    console.log(name);
+    console.log(course);
+    console.log(grade);
```
Comment (WhiskeyJack96) [inline features/components/app.js:42]: These can be cleaned up!

File: features/components/grade-table.js [hunk 2]
```diff
@@ -35,8 +38,8 @@ class GradeTable {
     for( var i = 0; i < gradesList.length; i++){
       var studentGrade = gradesList[i];
       sum = sum + studentGrade;
-      var average = sum/gradesList.length
+      var average = sum/gradesList.length;
```
Comment (WhiskeyJack96) [inline features/components/grade-table.js:41]: This can be done in the return statement!

File: features/components/app.js (hunk 1 above)
Comment (billy2anne) [reply features/components/app.js]: Done, removed console log functions.

File: features/components/grade-table.js (hunk 2 above)
Comment (billy2anne) [reply features/components/grade-table.js]: Made changes, can you check if its correct?

**Pull request merged**